transformations/emoji_icon_transformation/transformation.py

```diff
@@ -0,0 +1,201 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def convert(perturbed_text, dict1, dict2):
+    for k in dict1:
+        for s in dict1[k]:
+            if s in perturbed_text:
+                perturbed_text = perturbed_text.replace(
+                    s, random.choice(dict2[k])
+                )
+    return perturbed_text
+
+
+def emoji2icon(text, seed=42, max_outputs=1, emoji_to_icon=True):
+    random.seed(seed)
+
+    icons = {
+        "smiley": [
```

Comment:
I would suggest keeping these as a separate json file to make them easily expandable, rather than having these as part of the python script.

Reply:
Thank you @tongshuangwu for the suggestion. I have added the two json files in commit 55cb21bbad2ec536ee40a3648fd4a33151c03402